tests/test_jira_sync.py

```
@@ -81,6 +84,104 @@ def test_post_jira_worklog_rejects_naive_started_timestamp(self):
+            issue_key="KAN-1",
+            day="2026-04-20",
+            started=datetime(2026, 4, 20, 10, 0, tzinfo=timezone.utc),
+            seconds=7200,
+            projects=["Demo"],
+            source="commit",
+        )
+        fake_report = SimpleNamespace()
+        creds = JiraCredentials(
+            base_url="https://example.atlassian.net",
+            email="fake@example.com",
+            api_token="fake-token",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "api_token"

(S106)